waterdata/templates/states_counties.html

```diff
@@ -0,0 +1,89 @@
+{% extends 'base.html' %}
+
+{% if unit_cd %}{% set page_title = unit_data.name %}{% endif %}
+
+{% block content %}
+    <div class="usa-width-one-whole">
+        {% if http_code == 200 %}
+
+            {% if unit_data.name == 'US' %}
+                <h1>United States</h1>
+            {% else %}
+                <h1>{{ unit_data.name }}</h1>
+            {% endif %}
+                {% if unit_data.county_cd or unit_data.children %}
+                    <p>Select water monitoring location</p>
+                {% endif %}
+
+            {% if unit_data.name == 'US' %}
+                <table>
+                    <thead>
+                        <tr>
+                            <th scope="col">Unit Code</th>
+                            <th scope="col">State</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for state_cd in unit_data.children %}
+                            <tr>
+                                <th scope="row"><a href="{{ url_for('states_counties', state_cd=state_cd) }}">{{ state_cd }}</a></th>
+                                <td> {{ config.COUNTRY_STATE_COUNTY_LOOKUP.US.state_cd[state_cd].name }}</td>
+                            </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            {% endif %}
+
+            {% if unit_data.name != 'US' %}
+                {% if unit_data.county_cd %}
+                    <table>
+                        <thead>
+                            <tr>
+                                <th scope="col">Unit Code</th>
+                                <th scope="col">County</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for county_cd in unit_data.county_cd %}
+                                <tr>
+                                    <th scope="row"><a href="{{ url_for('states_counties', state_cd=state_cd, county_cd=county_cd) }}">{{ county_cd }}</a></th>
+                                    <td> {{ config.COUNTRY_STATE_COUNTY_LOOKUP.US.state_cd[state_cd].county_cd[county_cd].name }}</td>
+                                </tr>
+                            {% endfor %}
+                        </tbody>
+                    </table>
+                  {% endif %}
+            {% endif %}
+
+            {% if show_locations_link %}
+                 <a href="{{ url_for('county_station_locations', state_cd=state_cd, county_cd=county_cd) }}">Monitoring Locations</a>
+            {% endif %}
+
+
+        {% else %}
+            <h1>Error: HTTP {{ http_code }} -- State or county not found.</h1>
+        {% endif %}
+
+        {% if monitoring_locations %}
```

Comment:
I would put this before the else if line 63 to make clear that it's only shown if it's not a 404.